.github/workflows/build.yml

```diff
@@ -116,6 +125,13 @@ jobs:
       with:
         repository: ruby/ruby
         ref: ${{ needs.prepare.outputs.commit }}
+      if: matrix.name != '3.4-asan'
+    - name: Clone ruby (3.4-asan)
+      uses: actions/checkout@v4
+      with:
+        repository: ruby/ruby
+        ref: ${{ needs.prepare.outputs.commit_3_4_asan }}
+      if: matrix.name == '3.4-asan'
```

Comment:
Instead of another step can we instead use a conditional in `ref:`?
We're cloning ruby/ruby in both cases, just different commits.

Reply:
Fixed in a96283941b6523a2cb09f4cabb9db30f751e6a93